packages/editor/src/store/reducer.js

```diff
@@ -913,6 +913,29 @@ export function postLock( state = { isLocked: false }, action ) {
 	return state;
 }
 
+/**
+ * Post saving lock.
+ *
+ * When post saving is locked, the post cannot be published or updated.
+ *
+ * @param {PostSavingLockState} state  Current state.
+ * @param {Object}              action Dispatched action.
+ *
+ * @return {PostLockState} Updated state.
+ */
+export function postSavingLock( state = [], action ) {
```

Comment:
You changed the return shape for the lock actions, but the default is still an array.

Reply:
good point! addressed in https://github.com/WordPress/gutenberg/pull/10649/commits/e97e01a815fd3c50c4aebf65ae1e442a00948daf